github/authorizations.go

```diff
@@ -321,3 +335,65 @@ func (s *AuthorizationsService) Revoke(clientID string, token string) (*Response
 
 	return s.client.Do(req, nil)
 }
+
+// List the grants for a user. This will return one entry for each OAuth
+// application that has been granted access to the account, regardless of the
+// number of tokens an application has generated for the user.
+//
+// GitHub API docs: https://developer.github.com/v3/oauth_authorizations/#list-your-grants
+func (s *AuthorizationsService) ListGrants() ([]*Grant, *Response, error) {
```

Comment:
`golint` would say:

```
comment on exported method AuthorizationsService.ListGrants should be of the form "ListGrants ..."
```

All three methods (ListGrants, GetGrant, DeleteGrant) need to be updated to follow that style.

Source: https://golang.org/doc/effective_go.html#commentary

> The first sentence should be a one-sentence summary that **starts with the name being declared.**